projects/angular-material-extensions/select-country/src/lib/db.ts

```diff
@@ -1,1224 +1,1426 @@
 import {Country} from './mat-select-country.component';
 
 export const COUNTRIES_DB: Country[] =
```

Comment:
@cedvdb may you check the latest version `3.0.1` and adapt your code into the multiple i18n dbs ?

Thank you for your work